### ISSUE TITLE
Error when importing XML BPMN 

### ISSUE THREAD

**Issue by chrismatix** [process-intelligence-solutions/pm4py#313]
I have created a BPMN XML 2.0 diagram using [this](https://cawemo.com/) online editor (file attached). When trying to import the code I get the following error:

```
~/.conda/envs/pm/lib/python3.9/site-packages/pm4py/objects/bpmn/importer/variants/lxml.py in parse_element(bpmn_graph, counts, curr_el, parents, incoming_dict, outgoing_dict, nodes_dict, nodes_bounds, flow_info, process, node, bpmn_element, flow, rec_depth)
    223         # bpmn_graph.set_process_id(process)
    224         for seq_flow_id in incoming_dict:
--> 225             if incoming_dict[seq_flow_id][0] in nodes_dict:
    226                 incoming_dict[(seq_flow_id)] = (nodes_dict[incoming_dict[seq_flow_id][0]], incoming_dict[seq_flow_id][1])
    227         for seq_flow_id in outgoing_dict:

TypeError: 'Task' object is not subscriptable
```

The file seems to be valid because I can import it with other tools, too. Just to be sure though I tried creating the same model with [bpmn.io](https://bpmn.io/). Here the import works.

[broken-bpmn.txt](https://github.com/pm4py/pm4py-core/files/7989480/broken-bpmn.txt)
[working-bpmn.txt](https://github.com/pm4py/pm4py-core/files/7989485/working-bpmn.txt)

Thank you very much for creating and maintaining this library.

**Comment by fit-alessandro-berti** [comment]
Dear christmatix,

Thanks for signaling. We have discovered the root cause of the issue, and release a fix in the upcoming release.

Have a nice day

Alessandro

**Comment by fit-alessandro-berti** [comment]
Dear christmatix,

We resolved the issue in the release 2.2.19.1

Have a nice day

Alessandro
